src/modules/deno.rs

```diff
@@ -0,0 +1,108 @@
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::deno::DenoConfig;
+use crate::formatter::StringFormatter;
+
+/// Creates a module with the current Deno version
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("deno");
+    let config = DenoConfig::try_load(module.config);
+    let is_deno_project = context
+        .try_begin_scan()?
+        .set_files(&config.detect_files)
+        .is_match();
+
+    if !is_deno_project {
+        return None;
+    }
+
+    let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+        formatter
+            .map_meta(|var, _| match var {
+                "symbol" => Some(config.symbol),
+                _ => None,
+            })
+            .map_style(|variable| match variable {
+                "style" => Some(Ok(config.style)),
+                _ => None,
+            })
+            .map(|variable| match variable {
+                "version" => context
+                    .exec_cmd("deno", &["-V"])
+                    .and_then(|output| parse_deno_version(output.stdout.trim()))
```

Comment:
This calls `.trim()`, but later in the test you don't call `.trim()` before passing to `parse_deno_version`, is it needed?

Reply:
No, but I mean I'll add it.